acceptance/windows-service/test/integration/chef-windows-service/inspec/chef_windows_service_spec.rb

```diff
@@ -0,0 +1,58 @@
+only_if do
+  os['family'] == 'windows'
+end
+
+describe command("chef-service-manager") do
+  it { should exist }
+  its('exit_status') { should eq 0 }
+end
+
+describe service('chef-client') do
+  it { should_not be_enabled }
+  it { should_not be_installed }
+  it { should_not be_running }
+end
+
+describe command("chef-service-manager -a install") do
```

Comment:
Yeah it's a little unusual.. but it works very elegantly and simply to test the sequence of states.